Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -55,6 +62,18 @@ class UserScriptManager {
         return WKUserScript(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: false)
     }()
     
+    private let cookieControlUserScript: WKUserScript? = {
+        guard let path = Bundle.main.path(forResource: "CookieControl", ofType: "js"), let source: String = try? String(contentsOfFile: path) else {
+            log.error("Failed to load cookie control user script")
+            return nil
+        }
+        var alteredSource: String = source
+        let token = UserScriptManager.securityToken.uuidString.replacingOccurrences(of: "-", with: "", options: .literal, range: nil)
+        alteredSource = alteredSource.replacingOccurrences(of: "$<local>", with: "L\(token)", options: .literal, range: nil).replacingOccurrences(of: "$<session>", with: "S\(token)", options: .literal, range: nil).replacingOccurrences(of: "$<cookie>", with: "C\(token)", options: .literal, range: nil)
```

Comment:
Small nit, maybe break this into multiple lines (super long), and also `range` should have a default so I don't think you have to include it as nil.

```suggestion
        alteredSource = alteredSource.replacingOccurrences(of: "$<local>", with: "L\(token)", options: .literal)
                                     .replacingOccurrences(of: "$<session>", with: "S\(token)", options: .literal)
                                     .replacingOccurrences(of: "$<cookie>", with: "C\(token)", options: .literal)
```

Reply:
Done